root/etc/cont-init.d/50-config

```diff
@@ -110,15 +77,8 @@ if ! grep -q '#Removed lua' /config/nginx/nginx.conf; then
 fi
 
 # patch authelia-server.conf for CVE-2021-32637
-if ! grep -q 'if ($request_uri ~' /config/nginx/authelia-server.conf; then
-    sed -i '/internal;/a \ \ \ \ if ($request_uri ~ [^a-zA-Z0-9_+-=\\!@$%&*?~.:#'\''\\;\\(\\)\\[\\]]) { return 401; }' /config/nginx/authelia-server.conf
-fi
-
-# copy pre-generated dhparams or generate if needed
-[[ ! -f /config/nginx/dhparams.pem ]] && \
```

Comment:
Isn't this still needed?

Reply:
It's been moved into the base https://github.com/linuxserver/docker-baseimage-alpine-nginx/blob/80618475ed4d591f938db2e9b861c72274b0819d/root/etc/cont-init.d/13-nginx#L29-L31